README.md

```diff
@@ -78,6 +79,21 @@ To launch the interactive CLI, run the executable without any arguments.
 
 Run `ficsit help` to see a list of available commands.
 
+## Game Installations
+
+The app needs to know where the game is installed. On Steam, this will be
+
+    C:\Program Files (x86)\Steam\steamapps\common\Satisfactory
+
```

Comment:
- This section is missing mention of Linux
- That's the default steam path, but it depends on where the user installed it
- I would rather link to https://docs.ficsit.app/satisfactory-modding/latest/faq.html#Files_GameInstall than list them all here. If we do that, we need to add the mac location to the FAQ page.

Reply:
- I don't know what the path would be on Linux
- Nothing in this README even mentions that FAQ page, how would anyone even know to look there?
- Forcing a person to dig through an entire FAQ for the one piece of information they **need** to make this app work is overkill
- Fair point, I'll change the text to clarify this is only the default path

Reply:
"How would anyone even know to look there" because this readme would be changed to link them to it

"Forcing a person to dig through an entire FAQ" please note that the link I posted takes viewers directly to the relevant heading

Reply:
Added Mac path to FAQ in https://github.com/satisfactorymodding/Documentation/commit/8f7e73a1a25c197f0d804fd3633a0e388cfa25f8

Reply:
That FAQ doesn't tell me the default. Nor does it give me enough of a hint to work out what my own path would be if I had installed to a custom location without forcing me to go through the Steam UI.

Remember... we're talking here about folk with the technical literacy to both install to a custom location in the first place and to favour a command-line tool over SMM. For this audience in particular "here is the default path" is the only information they need.

Reply:
See https://github.com/satisfactorymodding/ficsit-cli/pull/61/files#r1562711449

Reply:
That appears to just be a link back to this PR

Reply:
What hint do you suggest using providing to work out a custom install location?

Reply:
> That appears to just be a link back to this PR

My bad, not sure why that link broke, here's a new one: https://github.com/satisfactorymodding/ficsit-cli/pull/61#discussion_r1562711449

Reply:
> What hint do you suggest using providing to work out a custom install location?

The default path **IS** the hint.

It tells me everything I need to know.

- It tells me I want a path pointing to the `Satisfactory` folder (and not, say, the `FactoryGame` folder underneath that). This is true even in a custom steam folder
- It tells me to look under `${HOME}/Library/Application Support/CrossOver/Bottles/<bottle name>/drive_c` on MacOS

For someone used to working with command line tools and comfortable doing so, that's all you really need.

Reply:
This PR, as it stands, is entirely self-contained with everything needed to build and run the app and install mods, without any need for external reference to an FAQ that is clearly written to hand-hold users with a lower degree of technical competence.

That's not a property I'm willing to sacrifice.

Reply:
I don't understand why you are implying that we want to reject this PR.

Regarding the FAQ's path being incorrect, I used the exact same path you used in your previous commit (https://github.com/satisfactorymodding/ficsit-cli/commit/6d70a5ce606c29bf7030ba9295a04f5274ba6509#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R90). I have since updated it to the new path you made in another commit (https://github.com/satisfactorymodding/ficsit-cli/commit/2a45c6acbb59933ca82cdd9a1cdb624216b02ed5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R93) after I made the original FAQ change. 

Users with the technical competence to set up custom install paths also have the patience to read a brief documentation passage and the understanding that others may not possess the same knowledge.